plugins/nessus.rb

```diff
@@ -1136,7 +1136,14 @@ def cmd_nessus_db_scan_workspace(*args)
         end
         targets.chop!
         print_status("Creating scan from policy #{policy_id}, called \"#{name}\" and scanning all hosts in #{framework.db.workspace.name}")
-        scan = @n.scan_create(policy_id, name, desc, targets)
+        et=Hash.new
```

Comment:
I've used only the same syntax as in `cmd_nessus_scan_new` and `cmd_nessus_db_scan` ;-).

```
root@kali:/usr/share/metasploit-framework/plugins# grep -n -A2 "et=Hash.new" nessus.rb 
971:          et=Hash.new
972-          et['enabled']=false
973-          et['launch']='ONETIME'
--
1080:        et=Hash.new
1081-        et['enabled']=false
1082-        et['launch']='ONETIME'
--
1139:        et=Hash.new
1140-        et['enabled']=false
1141-        et['launch']='ONETIME'
root@kali:/usr/share/metasploit-framework/plugins# 

```